src/Heading/docs/index.story.js

```diff
@@ -24,15 +32,37 @@ export default {
     ellipsis: false,
   },
 
-  examples: (
-    <div>
-      <CodeExample title="Ellipsis Example" code={EllipsisExampleRaw}>
-        <EllipsisExample />
-      </CodeExample>
-
-      <CodeExample title="Typography Example" code={TypographyExampleRaw}>
-        <TypographyExample />
-      </CodeExample>
-    </div>
-  ),
+  exampleProps: {},
+
+  sections: [
+    header(),
+
+    tabs([
+      tab({
+        title: 'Description',
+        sections: [
```

Comment:
Missing description block.
Suggestion:
```suggestion
        sections: [
          description({
            title: 'Description',
            text:
              'Headings are texts that to be used for titles or subtitles.',
          }),
```